src/tlo/methods/wasting.py

```diff
@@ -1,41 +1,1277 @@
+        """
+        Helper function to use the duration of the wasting episode to get date of outcome (recovery, progression,
+        or death)
+        :param person_id:
+        :return: date of outcome, which can be recovery to no wasting or progression to severe wasting from moderate
+        wasting; or recovery to moderate wasting or death due to severe wasting
+        """
+        df = self.sim.population.props
+        p = self.parameters
+        whz_category = df.at[person_id, 'un_WHZ_category']
+
+        # moderate wasting (for progression to severe, or recovery to no wasting) -----
+        if whz_category == '-3<=WHZ<-2':
+            # Allocate the duration of the moderate wasting episode
+            duration_mod_wasting = int(max(p['min_days_duration_of_wasting'], p['duration_of_untreated_mod_wasting']))
+            # Allocate a date of outcome (progression, or recovery)
+            date_of_outcome = df.at[person_id, 'un_last_wasting_date_of_onset'] + DateOffset(days=duration_mod_wasting)
+            return date_of_outcome
+
+        # severe wasting (for death, or recovery to moderate wasting) -----
+        if whz_category == 'WHZ<-3':
+            # determine the duration of SAM episode
+            duration_sev_wasting = int(max(p['min_days_duration_of_wasting'], p['duration_of_untreated_mod_wasting']
+                                           + p['duration_of_untreated_sev_wasting']))
+            # Allocate a date of outcome (progression, recovery or death)
+            date_of_outcome = df.at[person_id, 'un_last_wasting_date_of_onset'] + DateOffset(days=duration_sev_wasting)
+            return date_of_outcome
+
+    def population_poll_clinical_am(self, population):
+        """
+        Update at the population level other anthropometric indices and clinical signs (MUAC, oedema,
+        medical complications) that determine the clinical state of acute malnutrition. This will include both wasted
+        and non-wasted children with other signs of acute malnutrition.
+        :param population: population dataframe
+        """
+        df = population
+
+        # give MUAC measurement category for all WHZ, including well
+        # nourished children -----
+        for whz in ['WHZ<-3', '-3<=WHZ<-2', 'WHZ>=-2']:
+            index_6_59mo_by_whz = df.index[df.is_alive & (df.age_exact_years.between(0.5, 5, inclusive='left'))
+                                           & (df.un_WHZ_category == whz)]
+            self.muac_cutoff_by_WHZ(idx=index_6_59mo_by_whz, whz=whz)
+
+        # determine the presence of bilateral oedema / oedematous malnutrition
+        index_under5 = df.index[df.is_alive & (df.age_exact_years < 5)]
+        self.nutritional_oedema_present(idx=index_under5)
+
+        # determine the clinical acute malnutrition state -----
+        for person_id in index_under5:
+            self.clinical_acute_malnutrition_state(person_id=person_id, pop_dataframe=df)
```

Comment:
Yes, as per above, again see here: https://www.ncbi.nlm.nih.gov/books/NBK154454/

Reply:
I've checked this again, so it seems that all: oedema, WHZ, and clinical state of acute malnutrition are determined for all children under 5 y. old, only MUAC is determined for children 6-59 months, meaning it will be always assumed that children 0-5 months have normal MUAC. @timcolbourn, Is that a meaningful assumption?